src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -52,6 +54,11 @@ public void initialize() {
     loginProgressPane.setVisible(false);
     loginErrorLabel.managedProperty().bind(loginErrorLabel.visibleProperty());
     loginErrorLabel.setVisible(false);
+
+    // fixes #459
+    // "re" loads the font for the FAF logo and warning icons on the login page
+    // these are defined in style.css ".icon" but not used if WindowController.maximize() is called at stage initialization
+    Font.loadFont(LoginController.class.getResource("/font/dfc-icons.ttf").toExternalForm(), 10);
```

Comment:
I just realize that this font isn't loaded anywhere anymore, not even in the CSS? This was never intended :) You can load this font in the class `Main` like so:
```
    Font.loadFont(getClass().getResourceAsStream("/font/dfc-icons.ttf"), defaultFontSize);
```

And the Open Sans font should be loaded in the CSS file (I could swear it was there....). However, this, unfortunately, causes issues with SceneBuilder (doesn't render style anymore). I think it was something like this:

```
@font-face {
   src: url("@../font/dfc-icons.ttf");
}
```
or:
```
@font-face {
   src: url("/font/dfc-icons.ttf");
}
```

IIRC one worked in SceneBuilder, while the other worked in the program.